client/src/components/main/auth_modal/AuthModal.js

```diff
@@ -24,7 +26,17 @@ export default ({ background })=> {
 
   let headerContent, footer, form;
   if (formType === "login") {
-    headerContent = "Sign In";
+    headerContent = (
+      <div className="welcome-message">
+        <img
+          src="https://roastia.s3.us-east-2.amazonaws.com/roastia+logo-01.png"
+          alt="logo"
+          className="roastia-logo"
+        />
+        <h3> Sign in or register </h3>
+        <h3> to save your favorite coffee shops</h3>
```

Comment:
This is also very much not what I wanted to happen here, in the documentation for the `AuthLink` it states that.
> The component also accepts an optional notice boolean prop. If this prop is present the component will attach the notice "you must sign in before performing that action" will be attached to location.state.notice, we will have to actually render this on the modal if we want users to actually see it  

This notice should be conditionally rendered here, instead of always just stating that you must sign in or register to save shops. I guess it does make sense to explain the benefit of signing in but that does make the work I put into build that functionality into the `AuthLink` absolutely bunk.

Reply:
Then we could just render 'sign in' or 'register' based on the login/sign up panel